File: backend/src/main/scala/ch/countrygame/Classes/Manpower.scala
package ch.countrygame.Classes

import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}

case class Manpower(manpowerId: Int, manpowerType: Int, manpowerNumber: Double, countryId: Int) {
}
object Manpower {
  implicit val encoder = deriveEncoder[Manpower]
  implicit val decoder = deriveDecoder[Manpower]
}

File: frontend/src/app/models/manpower.ts
export class Manpower {
  manpowerId: number;
  manpowerType: number;
  manpowerNumber: number;
  countryId: number;

  constructor(manpowerId: number, manpowerType: number, manpowerNumber: number, countryId: number) {
    this.manpowerId = manpowerId;
    this.manpowerType = manpowerType;
    this.manpowerNumber = manpowerNumber;
    this.countryId = countryId;
  }
}
